src/inference/include/openvino/runtime/properties.hpp

```diff
@@ -232,7 +233,7 @@ inline std::istream& operator>>(std::istream& is, Priority& priority) {
  * @brief High-level OpenVINO model priority hint
  * Defines what model should be provided with more performant bounded resource first
  */
-static constexpr Property<Priority> model_priority{"OV_MODEL_PRIORITY"};
+static constexpr Property<Priority> model_priority{"MODEL_PRIORITY"};
```

Comment:
@wangleis note, that now this option named as in API 1.0. But for API 1.0 possible values are `MODEL_PRIORITY_LOW`, `MODEL_PRIORITY_HIGH`, etc. while for new API we have values `HIGH`, `LOW`, etc and it should be taken into account